src/applications/vaos/actions/newAppointment.js

```diff
@@ -62,12 +62,6 @@ export const FORM_FETCH_CHILD_FACILITIES =
   'newAppointment/FORM_FETCH_CHILD_FACILITIES';
 export const FORM_FETCH_CHILD_FACILITIES_SUCCEEDED =
   'newAppointment/FORM_FETCH_CHILD_FACILITIES_SUCCEEDED';
-export const FORM_FETCH_AVAILABLE_APPOINTMENTS =
```

Comment:
Renamed `FORM_FETCH_AVAILABLE_APPOINTMENTS` to `FORM_CALENDAR_FETCH_SLOTS` to match other calendar-related actions.